pkg/kv/kvserver/metrics.go

```diff
@@ -311,6 +311,18 @@ var (
 		Measurement: "Keys/Sec",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaAverageWriteBytesPerSecond = metric.Metadata{
+		Name:        "rebalancing.writebytespersecond",
```

Comment:
Should these still be prefixed with "rebalancing."? They seem generally useful, and the prefix feels non-intuitive.

Reply:
We could make a choice to rename them all - however qps/writes/reads etc are all against rebalancing.

I don't think it makes a difference here either way we go.